app/src/main/java/com/example/henzoshimada/feeltrip/FilterController.java

```diff
@@ -9,15 +9,61 @@
     private boolean mostrecentfilter;
     private boolean friendsonlyfilter;
 
-    public static void togglePastweek() {
+    private String emotionfilter;
+    private String keywordfilter;
 
+    public FilterController() {
+        pastweekfilter = false;
+        mostrecentfilter = false;
+        friendsonlyfilter = false;
```

Comment:
FilterController now holds all selected filters so we don't have to pass that data.